### ISSUE TITLE
Request: Ignore illegal cargo/equipment on the captured ship.

### ISSUE THREAD

**Issue by binaryspica** [endless-sky/endless-sky#10180]
### Problem Description

Well if you want to use the illegal cargo you have the chance to be fined, indeed, but... what if you didn't want to use that but the ship captured from the filthy pirates has it already? You can get caught by a random scan and got fined, and If it is an equipment you are automatically fined as well when you dock.

This is very annoying since I don't want to use it at all,and all I did was to fight against those who use this, despite they blame me for this.

### Related Issue Links

None I can aware of.

### Desired Solution

- At least before docking to a planet with a port(therefore it can be repaired) the captured ship should be exempt from any kind of fine for having illegal cargo/equipment, both for scanning and docked.
-If the docked port does not have an outfitter, you may have a choice to discard all the illegal cargo/equipment, or detach those equipments and move it to the cargo during docked or before take off, and the protection wears off.
-When you dock a planet with an outfitter, you are warned that you need to detach those equipments or you will be fined, and the protection wears off.

### Alternative Approaches

Perhaps either the government and/or the authority ships may offer to remove those equipments immediately or you would be fined, rather than simply fine you for having those equipments.

**Comment by TomGoodIdea** [comment]
An option to tell the government that the ship is captured and the illegal outfits were there before would fit nicely in the new hail panel in #6376, but it would still allow some exploitation as technically nothing prevents you from using those outfits (you could capture a ship, use its illegal outfits for some time and then say to ships scanning you that the outfits weren't yours).
Alternatively, we could allow dumping any cargo and outfits immediately after the ship is captured, but the player would need to know which outfits are considered illegal.

**Comment by binaryspica** [comment]
> An option to tell the government that the ship is captured and the illegal outfits were there before would fit nicely in the new hail panel in #6376, but it would still allow some exploitation as technically nothing prevents you from using those outfits (you could capture a ship, use its illegal outfits for some time and then say to ships scanning you that the outfits weren't yours). Alternatively, we could allow dumping any cargo and outfits immediately after the ship is captured, but the player would need to know which outfits are considered illegal.

Indeed, it helps when we have a mark when we saw the illegal cargo.

Asides, one of the other issue on here asks about SLAVE, which must be illegal(or even worse), but I don't think that it is a good idea to 'dumping' this for obvious reason.

**Comment by Hurleveur** [comment]
I think this is a fine (pun intended) thing to have, so long as it isnt an atrocity. Capturing is pretty strong, and if you need to pay an extra 1M for a pirate ship it's probably still worth it anyway (or 10M for a world ship). I don't think it happens often either, if at all, for pirate ships?

In general for atrocities and fines, I think we need to look at the precise cases where it happens in game, and if it makes sense to fine/arrest you.

**Comment by Zitchas** [comment]
I don't think the player should ever get a free pass.

 "Honest, officer, I just found this ship drifting out there and it had all this nerve gas on it. It isn't mine, I'm just bringing it back to port for safety." 

Should lead to "Oh, if it isn't yours, then here is 100k reward for bringing in a derelict ship, and we will take the ship and everything in it off your hands for proper disposal." Yoink, no more ship.

If the player wants to keep the ship as legally theirs, then they get to take the good with the bad. 

Now, having someway to ditch outfits, that would be good. Some things, like grenades, should be as easy to get rid of as cargo. Other things that are actually functionally installed things in ships... Those should be harder. Might need to find an outfitter that didn't care about legality to help get rid of things... (This is why pirate Outfitters are a thing, right?)

**Comment by petervdmeer** [comment]
> I don't think the player should ever get a free pass.

I feel that the nicest solution is to make the choice _at the start of a boarding session_, before the ship is even captured. Not a big UI interaction, but just one checkbox; `neutralize illegal outfits after capture`, which the player checks or doesn't check.

If the player checks this checkbox, then Nerve Gas gets transformed into chemicals (cargo), and illegal automated defense systems (do we have those?) get transformed into electronics.

The checkbox should just remember the value from last time, so if the player chose to keep the illegal outfits last time, then the default is to keep it for the next run as well, while if the player choose to neutralize the illegal outfits last time, then the default is to also do that for the next run.

**Comment by Zitchas** [comment]
Well, nerve gas isn't that easy to dispose of, and converting alien biomachinery into electronics is the sort of thing that should need (at a minimum) a recycling factory. If anything, I'd like to see a smattering of pirate ships have nerve gas on them as a deterant, as well as an incentive to use an outfit scanner to pick targets that don't have illegal outfits if that's a concern for said player/captain...

In other words, people already have a choice, I don't see a need to remove all the consequences from taking the choice. Nor do I think removing a bit of work (aka finding a target that lacks illegal outfits if one is going to need to get the ship back through human-law-abiding space) is beneficial.

**Comment by petervdmeer** [comment]
> Well, nerve gas isn't that easy to dispose of, and converting alien biomachinery into electronics is the sort of thing that should need (at a minimum) a recycling factory. If anything, I'd like to see a smattering of pirate ships have nerve gas on them as a deterant, as well as an incentive to use an outfit scanner to pick targets that don't have illegal outfits if that's a concern for said player/captain...
>
> In other words, people already have a choice, I don't see a need to remove all the consequences from taking the choice. Nor do I think removing a bit of work (aka finding a target that lacks illegal outfits if one is going to need to get the ship back through human-law-abiding space) is beneficial.

Good points! I can agree with almost all of this, and my previous proposal was not that good. I however still feel that the example from **SomeTroglodyte** about getting a fine for a ship that was captured 5 seconds ago, while you are still in the middle of a fight with hostiles, together with the government officials that want to fine you, seems a bit odd. The government officials in that system do know that the ship was captured and not originally yours (since they are fighting by your side, in the same system).

Getting a fine upon landing however feels okay to me; just like you said; the player can choose where to land and if the player chooses to land on a port where the authorities care, then that is the players choice. (It might help if the fine message specifies which outfits were found, and on which ships, I think it currently doesn't. And it might also be nice if the outfits get confiscated there and then.)

I'm a bit mixed on getting a fine just after jumping out of very hostile territory. The ship has not been repainted yet into the players colors and the signs of battle should still be visible on the ship. We should on the other hand not make it too easy for the player to transport captured ships long distances through law-abiding space.

My new proposal therefore is:
Don't fine ships that were captured since the last time the player landed/launched/jumped, except when those ships made 2 jumps by themselves. (And we should also fine ships that were send ahead by the player using the map-move-function, and ships that land to refuel on a law-abiding port.)

**Comment by binaryspica** [comment]
> Really? Am I missing some not-displayed secret key binding?

I wonder that is there something missing? Since there is nothing we could missing in the first place.

**Comment by tekker2234** [comment]
> Well, nerve gas isn't that easy to dispose of, and converting alien biomachinery into electronics is the sort of thing that should need (at a minimum) a recycling factory. If anything, I'd like to see a smattering of pirate ships have nerve gas on them as a deterant, as well as an incentive to use an outfit scanner to pick targets that don't have illegal outfits if that's a concern for said player/captain...
> 
> In other words, people already have a choice, I don't see a need to remove all the consequences from taking the choice. Nor do I think removing a bit of work (aka finding a target that lacks illegal outfits if one is going to need to get the ship back through human-law-abiding space) is beneficial.

I agree that it could incentivise using an outfit scanner. However, I would personally advise against taking this as a good thing. As it stands now, having a no-questions-asked fine for capturing a ship with illegal outfits is more of a punishment for not using the outfit scanner rather than a positive benefit for using it (you are literally getting fined, which is a criminal punishment). That creates a negative feeling around not using an outfit scanner rather than a positive feeling around using it, making the experience worse on net for the player (worse when they are not using it, the same as capturing ships with no chance of having illegal outfits when not using it).
Also for a player who encounters this for the first time this will feel very much like a "gotcha" type scenario, where the game leads them into a situation where they are punished for making an error that they were incentivized to make. (you are incentivized to capture pirates because it pays better than looting them. Now you have just been punished for doing the thing that the game is effectively telling you to do.) 

Personally I would be in favor of adding an option to destroy or throw away (into space) the illegal outfits of ships you capture. I feel like a good middle of the road option would be something like a warning when you capture a ship that has illegal outfits and then a time limit, like what Petervdmeer proposed, where if you don't get rid of your captured illegal outfits/cargo within a few jumps or before landing at a law abiding port (perhaps a thing where you surrender it to the authorities) then you can get fined.

Maybe that could also still work to incentivize scanners, have some kind of smallish reward for turning over the illegal items and open yourself up to fines if you don't. Maybe make it where the first time you land with an illegal item on a ship you have captured you get a dialogue option of turning it in for a reward or keeping it (to show the player the mechanic). I would then also have it be an option that the player can change in the settings so it doesn't become an annoying popup every time you land. Maybe also make it where the reward is less then you would get for selling it so there is some incentive to risk the fine if you don't want the outfit.

**Comment by binaryspica** [comment]
Not to mention that some ships that always have the illegal equipments. If I remember correctly, many Korath ships does, for many big Korath ships are have some microbot defense stations for all the variants, so there is no choice to avoid.

**Comment by binaryspica** [comment]
It would be easier to handle than marking each ships as seized until docking. Asking for something during the fly isn't something impossible either.

**Comment by Zitchas** [comment]
Perhaps also very relevant that maybe governments that the player has never visited and who are not allied with any governments the player has visited should have no means of directly and immediately draining the player's bank account. 


It kind of makes sense that all the human governments can do so, they all have access to the human bank. But some random alien I don't even know where they live , don't speak the same language, don't share a banking network... How is that fine going through, anyway? The story of it just doesn't make sense.

**Comment by waterhouse** [comment]
I vote for "Silently ignore illegal stuff on captured ships until your next takeoff."  That way I have the chance to use the outfitter to check for any nerve gas and dispose of it.  Since nerve gas is unplunderable, I don't think there is actually any way for the player to dispose of it (while keeping the ship) before landing (and even if it were plunderable, it would be really un-fun for the player to have to dig through the outfit lists every time for that rare entry).

Possible implementation: "When a ship is captured, it gets a grace marker which gives its illegal contents a pass; and all grace markers are removed on takeoff (or, equivalently, removed upon landing, directly _after_ the scan for illegal stuff)."  Lore could be that the debris from the disabling, and/or from the boarding party's laser rifle battle, on freshly captured ships confuses the scanning sensors.

**Comment by binaryspica** [comment]
> By the way, I'm now playing with a trivial 2-byte data only workaround: comment out the `unplunderable`:
> 
> ... this way, if I get fined, I can tell myself it's my own fault, I **did** have the option to dump. So far feels a little wrong of course, but still better than vanilla.

The problem is some of those are defensive devices attached on the ship itself. Nerve gas is the weapon(although it's for offensive), so do microbot defense system. For example it's possible to strip down all the defense systems on the Korath ships before the boarding begins, making its presence moot. It's funny that we can strip down all the normal stuffs without had a boarding fight at all, though, but that's an another problem.

Also, if you had the ships by win the boarding, you have a ship with microbot defense system anyways. And in this time the chance to loose the illegal items through the void is lost forever until docking with an outfitter. And the other escorts around the ships may grab this too, forcing you a double check for this.

**Comment by ravenshining** [comment]
> yup, still taking a lot of fun away. Visit the Korath exiles - you are attacked immediately. Capture one of their vessels that have Microbot defenses - no chance to dump them before, during, or after boarding.

Slaughtering civilians shouldn't be incentivised any more than tributing planets.  And how exactly are you supposed to just "dump" a homicidal airlock?  Maybe you were able to disable them on the way in and how have control via the ship's computers, but tamper with them too much, and you'll either wind up digested or venting part of the ship to vacuum.

**Comment by ravenshining** [comment]
Personally, I think it would be good to have a note in the UI when attempting to capture: "This ship has illegal outfits that may subject you to fines if you proceed."   IRL, if I were to find something highly illegal, I'm not going to want my fingerprints all over it - tip off the police to let them know it's there and steer clear.

As for nerve gas specifically, this might be mitigated if we had the ability for boarding weapons to be consumable.  Every round of combat, if a grenade was used in the calculation, a grenade is discarded.  If you can survive all a pirate's nerve gas... you've earned the ship.

Having the item confiscated could be a good mechanic if that's settable on a per-outfit basis separate from illegality (yes for nerve gas, no for microbots).  You'll still get fined of course for the disposal fees, but that's Republic bureaucracy for you.

**Comment by MrCauli** [comment]
> An option to tell the government that the ship is captured and the illegal outfits were there before would fit nicely in the new hail panel in [#6376](https://github.com/endless-sky/endless-sky/pull/6376), but it would still allow some exploitation as technically nothing prevents you from using those outfits (you could capture a ship, use its illegal outfits for some time and then say to ships scanning you that the outfits weren't yours). Alternatively, we could allow dumping any cargo and outfits immediately after the ship is captured, but the player would need to know which outfits are considered illegal.

They literally blabber endlessly about how "nerve gas is illegal" in the chat

**Comment by binaryspica** [comment]
We don't murdering people, ever; we just dispatch the pirate threat. All we expect is a thanks, nothing else, for who will be so hates somewhone who volunteered to clean the road?

Also remember that the public order of the human sphere of Endless Sky universe is dangerous and everyone who made a space flight are expected to arm themselves and deal with those threat by themselves, so clear some vermins wouldn't hurt your reputation.

**Comment by MrCauli** [comment]
> On a related tangent, I find it kind of odd how accepting the general population is of the player murdering people. So what if they're a criminal, that really isn't a good excuse for "Hey, I cheerfully murdered sixty people so I could steal their ship."
> 
> Most governments frown on that sort of thing since every once in a while someone murders a crew that _weren't_ wanted murderers and takes their ship. Sometimes they even get it into their heads to murder the crew of a _military_ ship, somehow mistaking it for pirates. Weird how that can happen...

Hunting pirates literally raises your reputation.

**Comment by slchorne** [comment]
I think the 2 byte fix from @SomeTroglodyte is the best: Add it to the plunder list

If **nerve gas** is an outfit you can sell, then you can find it on the ship, so: find it when plundering and have an option to dump it into space (move to cargo? dump? Ah... maybe the issue is hand to hand outfits can't become cargo?)
 
And add in a "This ship has illegal outfits..." dialog sometime before or after capturing the ship. At least give me an option to dump it before landing.

And yes, pay attention to the outfit scanner, but you are still stuck with the stuff until you land at an outfitter

I've had cases where I lost too many crew during the cap and had to land on the nearest planet - one that didn't have an outfitter, or not enough fuel to get to a system with an outfitter, so you don't just get 1 fine, but will keep getting fined until you make it to an outfitter to dump the nerve gas

Otherwise we're looking down the barrel of a more complicated "Just park this escort in space, don't land it" type enhancement

**Comment by TheGiraffe3** [comment]
Destroying pirates does in fact raise your reputation with the major human factions.

------

How easy would it be to put H2H outfits from captured ships into cargo automatically? Something like "we don't trust defenses, especially automated ones, on this newly-acquired ship"?

**Comment by waterhouse** [comment]
> Then I'd be curious where that is implemented. Haven't found anything else.

First, data files show significant negative attitudes toward the Pirate government from, for example, the Republic government:
```
government "Republic"
        swizzle 0
        color "governments: Republic"
        "crew attack" 1.2
        "crew defense" 2.2
        
        "player reputation" 2
        "attitude toward"
                "Alpha" -.3
                "Merchant" .25
                "Militia" .1
                "Pirate" -.3
```
This is relevant in the "Offend" procedure:
```
// Commit the given "offense" against the given government (which may not
// actually consider it to be an offense). This may result in temporary
// hostilities (if the even type is PROVOKE), or a permanent change to your
// reputation.
void Politics::Offend(const Government *gov, int eventType, int count)
{
...
        for(const auto &it : GameData::Governments())
        {
                const Government *other = &it.second;
                double weight = other->AttitudeToward(gov);
...
                if(count && abs(weight) >= .05)
                {
                        // Weights less than 5% should never cause permanent reputation
                        // changes. This is to allow two governments to be hostile or
                        // friendly without the player's behavior toward one of them
                        // influencing their reputation with the other.
                        double penalty = (count * weight) * other->PenaltyFor(eventType, gov);
                        if(eventType & ShipEvent::ATROCITY && weight > 0)
                                Politics::SetReputation(other, min(0., reputationWith[other]));

                        Politics::AddReputation(other, -penalty);
                }
```
The abs(weight) shows it's clearly deliberate that the "weight" can be negative, in which case the "penalty" is negative, and AddReputation gets a positive second argument.  And PenaltyFor will ultimately pick numbers from something like this (governments.txt is full of overrides, but this is the default):
```
// Default constructor.
Government::Government()
{
        // Default penalties:
        penaltyFor[ShipEvent::ASSIST] = -0.1;
        penaltyFor[ShipEvent::DISABLE] = 0.5;
        penaltyFor[ShipEvent::BOARD] = 0.3;
        penaltyFor[ShipEvent::CAPTURE] = 1.;
        penaltyFor[ShipEvent::DESTROY] = 1.;
        penaltyFor[ShipEvent::SCAN_OUTFITS] = 0.;
        penaltyFor[ShipEvent::SCAN_CARGO] = 0.;
        penaltyFor[ShipEvent::PROVOKE] = 0.;
        penaltyFor[ShipEvent::ATROCITY] = 10.;

```
Interestingly, you get a reputation increase for assisting a ship, and this is also implemented via a negative penalty.  I guess it follows that, for assisting a Pirate ship, you would actually get a reputation negative with the governments that hate Pirates.

**Comment by binaryspica** [comment]
Anyway, although I cause some collateral damage what I don't know, but aside that I killed no human at all. Destroying pirate isn't killing human at all in the first place. It's not the human, you know. It's which killing people, that must be eliminated - and that's why I don't see them as human being at all.

Personally I am proud that I have very, very low relationship with pirates and korath exiles. Especially when eats the worldship as breakfast and confirm kill the disabled ones. :p

**Comment by brendanjones** [comment]
Came to Github to whinge about this. Capturing marauder ships with nerve gas risks fines, which is annoying especially when you're doing your first low-end marauder missions. 

Along with simply reducing the prevalence of marauders through https://github.com/endless-sky/endless-sky/issues/7763, I would like to see the ability to use the player/ship info panel to dump non-structural outfits from ships while in flight. That way it's on the player to make sure their fleet is 'legal' before landing on a planet.

Is this proposed in another issue? I can't find it anywhere but surely it's been proposed before ...

**Comment by Quotatron3000** [comment]
> I would like to see the ability to use the player/ship info panel to dump non-structural outfits from ships while in flight. That way it's on the player to make sure their fleet is 'legal' before landing on a planet.

I think there should be a button next to 'Dump Cargo' (or maybe even a hotkey swap, like holding Ctrl changes the 'Dump Cargo' to 'Dump Illegals'). This isn't even purely for preventing do-gooders from receiving fines, as a smuggler sometimes you just gotta throw out the illegal goods to avoid being incriminated.

Maybe even turn it into an outfit that enables this feature, called the Jettison Hatch, only available at pirate Outfitters. Any captured pirate ship carrying nerve gas should also carry the Jettison Hatch.

**Comment by mikeloeven** [comment]
Honestly I would say after completing the Free Worlds arc or in the future joining the Navy you should have a dialogue option to Seize Contraband when capturing a ship. Instead of a fine the item goes into cargo as a generic item called Seized Contraband which will not trigger the fine but also renders it unusable ie you cant install or convert it back to its original item type. When landing at a port this is removed and you gain a small reward from the authorities this seems to be the best lore friendly way to handle the issue. Basically play it out as the player acting in an official enforcement capacity.

**Comment by ThrawnCA** [comment]
What if the on-capture message that says "X crew members were transferred" added a note, "They found illegal [cargo or outfit]! You may wish to dump this before entering systems with high security."

Dumping can be achieved from the 'I' dialog.

Edit: Ah, apparently you can't dump H2H from that dialog. Maybe you should be able to?

**Comment by ThrawnCA** [comment]
I just encountered a particularly bizarre form of this, where I was fined a million credits by the Remnant government, _in human space_, because I had picked up a Remnant ship as part of a rescue mission, which then followed me to human space, where I captured a pirate ship while it was present and it proceeded to scan me (as the Remnant do) before I could land and sell the ship.

Given that the Remnant ship in question watched the pirate ship change ownership, _and_ that we're out of Remnant jurisdiction, _and_ I'm in the the middle of saving their lives, that seems out of place.

**Comment by ThrawnCA** [comment]
It gets worse since there doesn't appear to be a way to remove illegal hand-to-hand outfits even while landed, if there's no outfitter.

Surely there should be a way to simply throw nerve gas out the airlock as soon as you board and capture.

**Comment by brendanjones** [comment]
What about scaling the nerve gas fine? https://github.com/endless-sky/endless-sky/discussions/12320 As I wrote there:

> What about scaling the fine based on the amount of the nerve gas you have? For example, 50,000 credits per nerve gas. This means at 20 nerve gas you'd get a 1 million fine, 50 nerve gas a 2.5 million fine. In other words, holding enough nerve gas for all crew on a warship would have real consequences. This would help so make the capturing mechanic somewhat harder (https://github.com/endless-sky/endless-sky/discussions/9949)

> This would also solve https://github.com/endless-sky/endless-sky/issues/10180 to a satisfactory degree, because Marauder ships have <10 nerve gas. So getting fined for 2 nerve gas (for example) would be a 100K fine. That's nothing to any player capturing Marauder ships.

> If https://github.com/endless-sky/endless-sky/issues/12307 was implemented, this scaled fine could also be translated to a scaled reputation reduction.

**Comment by waterhouse** [comment]
Come to think of it, the player is not able to _use_ nerve gas (or any other hand-to-hand weapons) on escort ships.  Therefore, nerve gas on escort ships is pretty much guaranteed to be on a captured ship that the player hasn't cleaned out yet.  "Keeping nerve gas on your escorts" is not a viable player strategy that needs to be punished.

I suggest making the fine only happen if the nerve gas is on the player's ship.  Lore could be that the nerve gas on your ship is in a "prepared" state, ready to be used at a moment's notice, which makes it emit small but distinctive and scannable traces; whereas nerve gas on escort ships is kept at -200° C and looks like cans of liquid nitrogen on scanners.

**Comment by ThrawnCA** [comment]
Gas kept on a Marauder ship is going to be ready for use, though, and the case we're talking about is where you're scanned seconds after capture. 

I still think the right solution is adding the ability to dump it during flight, ideally with a warning during the capture process that it exists. You can't uninstall regular outfits in flight, but you should be able to dump H2H outfits that take up zero space and zero mass, implying that they don't become part of the ship. 

Microbot defences would still need you to land at an outfitter first. Could there be a mechanic whereby the player provides their sensor logs to prove that the ship in question was captured and they haven't landed anywhere with an outfitter since that time, thus establishing that they didn't install it themselves and haven't been able to remove it?

**Comment by davidchisnall** [comment]
I was hit by this.  I captured a ship and it had nerve gas.  I landed, immediate 1M fine.  Nothing in the UI before then told me I had anything illegal.  When I landed, I went through my outfits and didn't see anything illegal, I missed the nerve gas.  Landed somewhere else and was fined again.

It would be very nice if, at the very least, you were told that a ship you'd boarded *had* illegal outfits.  And, ideally, had a way of disabling them immediately and then having them removed at the next outfitter to avoid the fine.

**Comment by brendanjones** [comment]
Yup, I get that sentiment @davidchisnall. 

Is there a reason why H2H outfits are hidden when you board a ship? If not, is there a reason why unplunderable outfits could not be shown but grayed out or something so that you can’t select them to steal them from the ship? And maybe illegal outfits could have an icon (or just the word ‘illegal’) next to them, denoting their illegality?

This would help solve the problem of people being clueless about carrying illegal outfits in their fleet.

**Comment by ThrawnCA** [comment]
Okay, I've kludged together a commit that adds a warning message upon capturing a ship with nerve gas. Someone who understands the codebase better could probably extend it to cover other illegal outfits, illegal cargo, etc (I haven't touched the C++ code before, only mission data).

Any feedback is welcome.

Edit: I managed to build and playtest it:

<img width="904" height="359" alt="Image" src="https://github.com/user-attachments/assets/dddcfcdd-ce73-4f79-90a8-e28fd462414f" />

**Comment by ThrawnCA** [comment]
Even if #12324 is approved, we still should have an option to take action sooner, eg being able to dump nerve gas mid flight.

Also, the one day grace period after being fined should potentially also apply after capturing a ship. Fundamentally, when you have just finished taking control of a ship and haven't yet had a chance to go anywhere or do anything with it, you shouldn't be fined for what was installed on it.

**Comment by ThrawnCA** [comment]
It would be more work to implement, but what if being scanned and found with contraband, instead of issuing an immediate fine, gave the player a period in which to make their case? Perhaps with a quest to go to a specific planet and report to Republic authorities there.

- If you turn up, and you have the logs to prove that you captured the contraband rather than buying it yourself, then the items are confiscated, you get a pat on the back for stopping pirates, and there are no further consequences.
- If you turn up with the contraband and you don't have any such excuse, then it's confiscated and you get a fine.
- If you turn up _without_ the contraband, then you're assumed to have passed it on or sold it, and you get a much larger fine, and possibly a reputation hit. Compare #12307
- If you don't turn up, then you get a large reputation hit.

**Comment by BobbyCobby** [comment]
> Perhaps with a quest to go to a specific planet and report to Republic authorities there.

Personally, I capture pirate ships constantly, so I think this happening every time my target has Nerve Gas would get irritating. But once in a while would be fun!

**Comment by ThrawnCA** [comment]
> Personally, I capture pirate ships constantly, so I think this happening every time my target has Nerve Gas would get irritating.

Not every time, just every time you get scanned before you can get rid of the nerve gas. The current situation is that you would be immediately fined a million credits, which IMO is even more irritating. If you aren't currently getting loads of fines, then you wouldn't get loads of quests.

Also, standard pirates don't have nerve gas, it's typically a Marauder thing. So you could still capture lots and lots of pirates without risking it, just be careful of quests and bounty hunters.

**Comment by mikeloeven** [comment]
Just going to put it out there I long since used a plugin to disable all of this once the game gives you nukes you cant really argue about war crimes anymore
